projects/fw_102_hw/src/main.c

```diff
@@ -0,0 +1,36 @@
+#include <stdint.h>
+#include "interrupt.h"
+#include "log.h"
+#include "soft_timer.h"
+#include "wait.h"
+// not sure how to make it work with struct inside main()
+// would appreciate feedback!

```

Comment:
Ah, by `In your main function, create a counters struct`, the homework just means to create a variable of your struct type inside main (which you've done with `count`), not put the struct definition inside, which indeed wouldn't work. So you've done it correctly!